Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -7,7 +7,7 @@
         "version": "KqlParameterItem/1.0",
         "query": "",
         "crossComponentResources": [
-          "{Subscription}"
+          "value::selected"
```

Comment:
this part is interesting, this section should be "empty" for parameters steps.  yet another little thing to look into

Reply:
yup I noticed that